.github/workflows/build.yml

```diff
@@ -83,15 +77,9 @@ jobs:
           - macos-latest
           - windows-latest
         java_version:
-          - 7
           - 8
-          - 9
```

Comment:
Do we really need to drop all the tests for other JDK versions? I would think dropping 7 is good enough? I know most are end of life, but I still like to make sure ArchUnit works if dropped in such a projekt :man_shrugging:

Reply:
Of course we wouldn't strictly have to drop support for the other JDK versions; I'll restore the end of life versions just in case.

Practically speaking I think it'd be quite rare for an issue not to show up in 8, 11 & 17, while still posing issues in say 10 or 14. And even if that would be the case, I imagine the recommended change would be to up- or downgrade to a supported JDK. Comparatively, [ASM seems to only test on JDK 11](https://gitlab.ow2.org/asm/asm/-/blob/master/.gitlab-ci.yml#L1), and any issues there are likely to cascade here.

Reply:
Yes, it might be a little paranoid, but you never know :joy: The part about the classpath scanning is a little tricky and has sometimes changed drastically.

Reply:
No problem at all; restored all the outdated JDKs and even added in 16 just to be sure.